src/misc/aviation/aircraftlights.h

```diff
@@ -147,8 +160,12 @@ namespace swift::misc::aviation
             SWIFT_METAMEMBER_NAMED(beaconOn, "beacon_on"),
             SWIFT_METAMEMBER_NAMED(navOn, "nav_on"),
             SWIFT_METAMEMBER_NAMED(logoOn, "logo_on"),
-            SWIFT_METAMEMBER(recognition, 0, DisabledForJson), // disable since JSON is used for network
-            SWIFT_METAMEMBER(cabin, 0, DisabledForJson) // disable since JSON is used for network
+            SWIFT_METAMEMBER_NAMED(recognitionOn,"recognition_on"), 
+            SWIFT_METAMEMBER_NAMED(cabinOn, "cabin_on"), 
+            SWIFT_METAMEMBER_NAMED(wingOn, "wing_on") 
+            //SWIFT_METAMEMBER(recognitionOn, 0, DisabledForJson), // disable since JSON is used for network
+            //SWIFT_METAMEMBER(cabinOn, 0, DisabledForJson), // disable since JSON is used for network
+            //SWIFT_METAMEMBER(wingOn, 0, DisabledForJson) // disable since JSON is used for network
```

Comment:
Just to keep this as a note in this PR. Enabling these attributes should be discussed with the other client devs.